eZ/Publish/Core/FieldType/Tests/AuthorTest.php

```diff
@@ -561,8 +561,8 @@ public function provideDataForGetName(): array
         );
 
         return [
-            [$this->getEmptyValueExpectation(), [], 'en_GB', ''],
-            [$authorList, [], 'en_GB', 'Boba Fett'],
+            [$this->getEmptyValueExpectation(), '', [], 'en_GB'],
+            [$authorList, 'Boba Fett', [], 'en_GB'],
```

Comment:
Why do we change the order of arguments generated by the provider and how is it related to PHP 8?

Reply:
Another deprecation warning: `PHP Deprecated:  Optional parameter [xxx] declared before required parameter [yyy] is implicitly treated as a required parameter`, ref: https://www.php.net/manual/en/migration80.deprecated.php#migration80.deprecated.core.

Reply:
From what I see the changes are limited to test classes only. If it were otherwise, I would say see we should make the arguments mandatory instead and remove the default value (as it was impossible to use the optional parameter anyway - it always had to be specified).

That is, unless those test classes were used in other libraries, which I know might have been the case sometimes?

Reply:
>From what I see the changes are limited to test classes only

This is the main reason why I decided to just switch arguments order. I am not sure if we should care about providing any BC for tests anyway.

Reply:
+1